cli/main.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
Could you add an initial `README.md` to the `cli` directory and explain how people can build and start using the Oasis CLI?

Reply:
Added something basic.

Reply:
It looks very good, thanks!